scenes/stealth_level/game_logic.gd

```diff
@@ -0,0 +1,25 @@
+@tool
+extends Node
+
+@export var player_instantly_loses_on_sight: bool = false:
+	set = set_player_instantly_loses_on_sight
+@export_range(0.5, 3.0, 0.1, "or_greater", "or_less") var zoom: float = 1.0:
+	set = set_zoom
+
+@onready var enemy_guards: Node2D = %EnemyGuards
+@onready var camera_2d: Camera2D = %Camera2D
+
+
+func set_player_instantly_loses_on_sight(new_value):
```

Comment:
This is probably raising a warning because the return value is missing:
```suggestion
func set_player_instantly_loses_on_sight(new_value) -> void:
```